python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -126,6 +126,7 @@ def bake(
         set: Dict[str, str] = {},
         variables: Dict[str, str] = {},
         stream_logs: bool = False,
+        context: Union[str, None] = None,
```

Comment:
The issue I have with context is that `context` already exists: https://docs.docker.com/build/bake/contexts/ and it's slight different.

What about being explicit? Unless I'm missing something, we can just use "remote_definition", it's the term used in the docker documentation.

Reply:
Yeah `remote_definition` seems good, changed to that.

The interesting behavior with `context` is that by default, the contents of the `remote_context` become the value of Bake target `context`s (not `contexts`), [for example in the test definition](https://github.com/gabrieldemarmiesse/python-on-whales/pull/656/files#diff-e30722b601498d80a6278c8eb440452e25f646641ded295374cd13402ff58fd9R669). However, this is only the _default_; if the Bake file defines a `context`, it may be unrelated to this value.

TBH I think the whole idea of this Bake functionality needs some clarification/cleanup from Docker folks, but until that happens I think this is the best we can do!